compose-development.yml

```diff
@@ -8,12 +8,14 @@ shareddatadev:
     - /data
     # database
     - /database/data
+    # temporary results directory used as pipe between multiple docker forests
+    - /tmp/osmaxx-results:/results
   net: "none"
 
 sourcedev:
   build: docker/volume-mount
   volumes:
-    - osmaxx-py:/home/osmaxx/source
+    - ./osmaxx-py:/home/osmaxx/source
```

Comment:
I was wrong-ish: `./` **will** be needed in _future_ docker-compose versions to distinguish relative paths from named volumes.